homu/server.py

```diff
@@ -279,7 +290,10 @@ def github():
 
     return 'OK'
 
-def report_build_res(succ, url, builder, repo_label, state):
+def report_build_res(succ, url, builder, repo_label, state, logger):
+    logger.debug('build result %s: builder = %s, succ = %s, current build_res = %s',
+                 state, builder, succ, state.build_res)
```

Comment:
Beware, `state.build_res` can be HUGE as it contains all of the URLs that Buildbot has reported. Actually, it is a dictionary whose keys are builders, and values are URLs. If we're going to report this information, it would be better to limit its output to the keys of the dict only.